docs/src/formulation-details.md

```diff
@@ -0,0 +1,85 @@
+
+A handbook for developing an understanding of the principles of convex relaxation applied to optimal power flow is:
+- Taylor, J. A. (2015). Convex optimization of power systems (1st ed.). Cambridge University Press.
+
+## Notes on the mathematical model for all formulations
+PowerModels implements a slightly generalized version of the AC Optimal Power Flow problem from [Matpower](http://www.pserc.cornell.edu/matpower/), as discussed [here](./math-model.html).
+
+In the next subsections the differences between PowerModels' bus and branch models and those commonly used in the literature are discussed.
+Consideration is given to these differences when implementing formulations from articles.
+
+### Standardized branch model
+The branch model is standardized as follows:
+- An idealized (lossless) transformer at the from side of the branch (immediately on node $i$) with a fixed, complex-value voltage transformation (i.e. tap and shift)
+- Followed by a pi-section with complex-valued line shunt admittance, where the from and to side shunt can have different values
+- A branch is uniquely defined by a tuple $(l,i,j)$ where $l$ is the line index, $i$ is the from node, and $j$ is the to node.
+- Thermal limits are defined in apparent power, and are defined at both ends of a line
+- Each branch has a phase angle difference constraint
+
+Nevertheless, in the literature, the following can be observed:
+- The to and from side line shunts are equal
+- The line shunt admittance is a pure susceptance (equivalent to shunt conductance set to 0)
+- A branch in a grid without parallel lines is uniquely defined by a tuple $(i,j)$ where $i$ is the from node, and $j$ is the to node.
+- Thermal limits are defined in current (total or series), complex power limits are approximated as a regular polygon, ...
+- Thermal limits are defined only at the from (or to) side
+- Phase angle difference constraints are not included
+
+Furthermore, "lifted nonlinear cuts" are used to improve the accuracy of PAD constraints for all formulations in the lifted S-W variable space:
+- Coffrin, C., Hijazi, H., & Van Hentenryck, P. (2017). Strengthening the SDP relaxation of ac power flows with convex envelopes, bound tightening, and valid inequalities. IEEE Trans. Power Syst., 32(5), 3549–3558. <https://doi.org/10.1109/TPWRS.2016.2634586>
+
+
+### Standardized bus model
+The bus model is standardized as follows:
+- A bus defines a complex power balance for all the sets lines, generators, loads, bus shunts connected to it, i.e. one can define multiple load  and shunt  components on each bus $i$
+
+Nevertheless, in the literature, a simplified bus model is often used:
+- Only a single (aggregated) load per bus is supported
+- Only a single (aggregated) bus shunt per bus is supported
+
+
+
+
+## `ACPPowerModel`
```

Comment:
I think we can get a more maintainable solution to this by putting these links in the docstring when the type is defined, then autodoc will pick them up automatically. If you want to call out specific doc strings items at this location, there is a way to do that as well.

I would also consider adding a bibtex entry to the docstring.

Reply:
Great suggestion, that's a better approach indeed. This will take a bit of time and my agenda has been filled up for the rest of the week, so I'll re-visit later.

Reply:
Ok, I'll merge this now and you can add an issue on the update.